src/main/software/amazon/event/ruler/input/StringValueParser.java

```diff
@@ -0,0 +1,14 @@
+package software.amazon.event.ruler.input;
+
+import software.amazon.event.ruler.MatchType;
+
+/**
+ * @author Aliaksei Bialiauski (abialiauski@solvd.com)
+ */
+@SuppressWarnings("UseOfConcreteClass")
+public interface StringValueParser extends MatchTypeParser {
+
+  default InputCharacter[] parse(final String value) {
+    return this.parse(MatchType.NO, value);
```

Comment:
Needs documentation (and supporting unit tests) to understand the context and expected behavior.